### ISSUE TITLE
KB: links in error panel can't be followed

### ISSUE THREAD

**Issue by beyackle** [microsoft/BotFramework-Composer#6461]
## Describe the bug

The bottom error panel shows links to dialogs or files with bugs, but the links are only usable with the mouse, not the keyboard.

## Version

1.3.1

## Expected behavior

The link should be usable with the keyboard as well as mouse.

## Screenshots

Keyboard focus here is on the `appsettings.json` element. Clicking this with the mouse correctly leads to the Bot Settings page, but hitting Enter doesn't follow the link.
![image](https://user-images.githubusercontent.com/61990921/111547456-6d7fae00-8736-11eb-8d5c-df58e4bb67d9.png)
